tasks/sep/006-2-foundation-guardian-updates/SignFromJson.s.sol

```diff
@@ -0,0 +1,537 @@
+    }
+
+    function checkLivenessGuard() internal view {
+        console.log("Running assertions on the LivenessGuard");
+
+        require(livenessGuard.version().eq("1.0.0"), "checkLivenesssGuard-000");
+        require(livenessGuard.safe() == address(securityCouncilSafe), "checkLivenessGuard-100");
+
+        // Each owner was recorded as live when the Guard was deployed, or was recorded as live during execution of
+        // this runbook.
+        address[] memory owners = securityCouncilSafe.getOwners();
+        for (uint256 i = 0; i < owners.length; i++) {
+            uint256 lastLive = livenessGuard.lastLive(owners[i]);
+            require(lastLive == 1713416868 || lastLive == block.timestamp, "checkLivenessGuard-201");
+        }
+    }
+
+    function checkDeputyGuardianModule() internal view {
+        console.log("Running assertions on the DeputyGuardianModule");
+
+        require(deputyGuardianModule.version().eq("1.0.0"), "checkDeputyGuardianModule-000");
+        require(deputyGuardianModule.deputyGuardian() == address(foundationSafe), "checkDeputyGuardianModule-100");
+        require(deputyGuardianModule.safe() == address(securityCouncilSafe), "checkDeputyGuardianModule-200");
+        require(
+            deputyGuardianModule.superchainConfig() == address(proxies.SuperchainConfig),
+            "checkDeputyGuardianModule-300"
+        );
+    }
+
+    function checkSecurityCouncilSafe() internal view {
+        console.log("Running assertions on the SecurityCouncilSafe");
+        // The SecurityCouncilSafe and FoundationSafe should have the same set of owners
+        address[] memory councilOwners = securityCouncilSafe.getOwners();
+        address[] memory foundationOwners = foundationSafe.getOwners();
+        // require(councilOwners.length == foundationOwners.length, "checkSecurityCouncilSafe-200");
+        if (councilOwners.length != foundationOwners.length) {
+            // this should be silenced once the previous runbook is completed
+            console.log(
+                "\x1b[31m TEMP ERROR: checkSecurityCouncilSafe-100",
+                councilOwners.length,
+                foundationOwners.length,
+                "\x1b[0m"
+            );
+        }
+        for (uint256 i = 0; i < councilOwners.length; i++) {
+            // require(foundationSafe.isOwner(councilOwners[i]), "checkSecurityCouncilSafe-201");
+            if (!foundationSafe.isOwner(councilOwners[i])) {
+                // this should be silenced once the previous runbook is completed
+                console.log("\x1b[31m TEMP ERROR: checkSecurityCouncilSafe-101", councilOwners[i], "\x1b[0m");
+            }
+        }
+
+        // See sepolia.json for config values being verified:
+        // https://github.com/ethereum-optimism/optimism/pull/10224/files
+        // The SecurityCouncilSafe should have a treshold of 2
+        require(securityCouncilSafe.getThreshold() == 2, "checkSecurityCouncilSafe-301");
```

Comment:
This will start failing after #176

```suggestion
        // The SecurityCouncilSafe should have a threshold of 3
        require(securityCouncilSafe.getThreshold() == 3, "checkSecurityCouncilSafe-301");
```

Reply:
Removed the threshold checks since they fail anyway because in sims we override the threshold to 1